external/netstandard/netstandard.depproj

```diff
@@ -35,6 +35,12 @@
     <PackageReference Condition="'$(NETStandardVersion)' &gt;= 1.2" Include="System.Data.SqlClient">
       <Version>4.3.0</Version>
     </PackageReference>
+    <PackageReference Condition="'$(NETStandardVersion)' &gt;= 1.3" Include="System.Security.Cryptography.Cng">
```

Comment:
Without these entries I get errors that there is a netstandard1.3/1.4 ref but no implementation assembly.